Symlink downloaded artifacts

manics opened the pull request:
For example:
- `omego download py` behaves as normal
- `omego download py --sym auto` symlinks the unzipped download to `OMERO.py`
- `omego download py --sym /tmp/abc` symlinks the unzipped download to `/tmp/abc`

Closes #60 

Changes (since 2017-03-09):
- the default `OMERO-CURRENT` sym has been changed to `OMERO.server`, though `omego` will warn if the old sym is detected.
- rebased
- travis fixed

joshmoore commented:
NB: while we're here, should we go ahead and change `OMERO-CURRENT` to `OMERO.server`? (That's what you've been doing elsewhere anyway)

cc: @kennethgillen @sbesson

sbesson commented:
Assuming our roadmap is `0.5.0`, changing the default to `OMERO.server` makes complete sense to me.

manics commented:
Do you think that should be a separate PR (since it's clearly a breaking change)?

manics commented:
Also travis has a [new error](https://api.travis-ci.org/jobs/178851434/log.txt?deansi=true):
```
$ psql -c "create user omero with password 'omero';" -U postgres
CREATE ROLE

$ psql -c "create database omero with owner omero;" -U postgres
CREATE DATABASE

$ psql -c "select 1;" -U omero -h localhost omero
 ?column? 
----------
        1
(1 row)

(END)

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminated
```

joshmoore commented:
Travis issue:

```
The repository at ome/omego was not found.
```

Closing & re-opening.

manics commented:
Discovered and fixed a bug in the creation of symlinks when the artifact has already been downloaded and unzipped. Also fixed a test that was effectively `assert True` 😄
